public/app/features/explore/spec/queryHistory.test.tsx

```diff
@@ -153,10 +161,15 @@ describe('Explore: Query History', () => {
     starQueryHistory(1, ExploreId.left);
     await assertQueryHistoryIsStarred([false, true], ExploreId.left);
     await assertQueryHistoryIsStarred([false, true], ExploreId.right);
+    expect(reportInteractionMock).toBeCalledWith('explore_query_history_starred', {
+      queryHistoryEnabled: false,
+      newValue: true,
+    });
 
     deleteQueryHistory(0, ExploreId.left);
     await assertQueryHistory(['{"expr":"query #1"}'], ExploreId.left);
     await assertQueryHistory(['{"expr":"query #1"}'], ExploreId.right);
+    expect(reportInteractionMock).toBeCalledWith('explore_query_history_deleted', { queryHistoryEnabled: false });
```

Comment:
```suggestion
    expect(reportInteractionMock).toBeCalledWith('grafana_explore_query_history_deleted', { queryHistoryEnabled: false });
```

Reply:
Also for interactions with specific queries, I am wondering if we shouldn't have `grafana_explore_query_history_query_deleted`. But not sure. Just to distinguish between interaction with query and whole query history.

Reply:
I'll merge it like this. I've added `grafana_` prefix 👍